packages/kbn-securitysolution-io-ts-types/src/time_duration/index.test.ts

```diff
@@ -39,6 +39,17 @@ describe('time_unit', () => {
     expect(message.schema).toEqual(payload);
   });
 
+  test('it should NOT validate a TimeDuration of 0 length', () => {
+    const payload = '0s';
+    const decoded = TimeDuration.decode(payload);
+    const message = pipe(decoded, foldLeftRight);
```

Comment:
Nit: why use `pipe` to invoke a single function? Calling directly `foldLeftRight(decoded)` looks much simpler.